bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -7,7 +7,7 @@ from utils.gcp import bigquery_etl_query, gke_command
 
 {% set ns = namespace(uses_fivetran=False) -%}
 {% for task in tasks -%}
-{% if task.depends_on_fivetran != None and task.depends_on_fivetran|length > 1 and ns.uses_fivetran == False -%}
+{% if task.depends_on_fivetran != None and task.depends_on_fivetran|length > 0 and ns.uses_fivetran == False -%}
```

Comment:
pretty sure this is supposed to be `0`, because without it my dag is missing necessary imports